tests/integration/deploy/test_deploy_command.py

```diff
@@ -1579,3 +1579,39 @@ def test_deploy_with_language_extensions(self):
         )
         deploy_process_execute = self.run_command(deploy_command_list)
         self.assertEqual(deploy_process_execute.process.returncode, 0)
+
+    @parameterized.expand([("aws-serverless-function.yaml", "samconfig-capabilities-list.toml")])
+    def test_deploy_with_valid_config_capabilities_list(self, template_file, config_file):
```

Comment:
nit and non-blocking: I wonder if it makes more sense to add functional tests rather than integration tests. It seems to me that what we actually want to test is that the list in the toml config file is parsed correctly.